sdk/Context/Context.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Context;
```

Comment:
Hrmmmmm... That's a good question. It definitely doesn't belong in the API namespace, it is used *by* the SDK... but an entirely separate namespace might actually be good to distinguish it as an "outside dependency" of sorts. Honestly it could go in its own repo and get vendored in as well.

Reply:
Yeah I think either one would be okay.  I'd lean towards its own namespace for simplicity's sake now.  We could always take it out and vendor it later if need be.